index.js

```diff
@@ -181,3 +185,55 @@ AddonManager.addInstallListener({
     app.send('addon-install:download-failed', formatInstallData(install));
   }
 });
+
+function pingServer(title, addon) {
+  cookieManager.getCookiesFromHost('ideatown.dev', function(cookies) {
```

Comment:
Do we have a settings file where we can put the hostname, instead of hard-coding it like this?

If not, let's file a followup bug to extract this out.